From 6d7dd529725c2b71c4499858cd8fbcfbd69c3b19 Mon Sep 17 00:00:00 2001
From: Yu Jiahua <yujiahua1@huawei.com>
Date: Tue, 19 Oct 2021 18:01:18 +0800
Subject: [PATCH] sched: Add menuconfig option for
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6d7dd529725c2b71c4499858cd8fbcfbd69c3b19

 CONFIG_SCHED_OPTIMIZE_LOAD_TRACKING

hulk inclusion
category: bugfix
bugzilla: 175551 https://gitee.com/openeuler/kernel/issues/I4DDEL
CVE: NA

--------------------------------

Move CONFIG_SCHED_OPTIMIZE_LOAD_TRACKING into menu "Scheduler features"

Signed-off-by: Yu Jiahua <yujiahua1@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Chen Jun <chenjun102@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 init/Kconfig | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index cedc5ab247a5..29682cbb327c 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -774,6 +774,14 @@ config GENERIC_SCHED_CLOCK
 
 menu "Scheduler features"
 
+config SCHED_OPTIMIZE_LOAD_TRACKING
+	bool "Optimize scheduler load tracking"
+	default n
+	help
+	  Optimize scheduler load tracking, when load balance is not important
+	  in system, we close some load tracking in tick and enqueue or dequeue
+	  task, in this way, we can save some unnecessary cpu overhead.
+
 config IAS_SMART_HALT_POLL
 	bool "Enable smart halt poll"
 	default n
@@ -2085,13 +2093,6 @@ endmenu		# General setup
 
 source "arch/Kconfig"
 
-config SCHED_OPTIMIZE_LOAD_TRACKING
-	bool "Optimize scheduler load tracking"
-	default n
-	help
-	  Optimize scheduler load tracking, when load balance is not important
-	  in system, we close some load tracking in tick and enqueue or dequeue
-	  task, in this way, we can save some unnecessary cpu overhead.
 
 config RT_MUTEXES
 	bool
-- 
2.26.2

